TOOLS/dylib-unhell.py

```diff
@@ -157,6 +162,104 @@ def process_swift_libraries(binary):
     print(">> setting additional rpath for swift libraries")
     install_name_tool_add_rpath("@executable_path/lib", binary)
 
+def process_vulkan_driver_libraries(binary):
+    # driver serach paths https://github.com/KhronosGroup/Vulkan-Loader/blob/main/docs/LoaderDriverInterface.md#example-macos-driver-search-path
+    vulkanDriverSearchPaths = [
+        os.path.join(os.path.expanduser("~"), ".config/vulkan/icd.d"),
+        "/etc/xdg/vulkan/icd.d",
+        "/usr/local/etc/vulkan/icd.d",
+        "/etc/vulkan/icd.d",
+        os.path.join(os.path.expanduser("~"), ".local/share/vulkan/icd.d"),
+        "/usr/local/share/vulkan/icd.d",
+        "/usr/share/vulkan/icd.d",
+    ]
```

Comment:
Probably it's worth to add Homebrew path too (`$HOMEBREW_PREFIX/share/vulkan/icd.d`) or (and?) make it configurable.

Reply:
according to the vulkan-loader docs those are not part of the search list.

though the plan is to make bundling part of the macos action and provide the bundle as an artifact. so we do need support for homebrew anyway. not sure how to do it for now and didn't look into homebrew yet. though i think i will add this in another commit.